resources/tests/test_api.py

```diff
@@ -30,21 +30,21 @@ def test_1_create_index(self):
 
     def test_2_video_docs(self):
         doc_list = api.get_video_docs()
-        self.assertTrue(doc_list is not None and len(doc_list) > 10)
+        self.assertEqual(len(doc_list), 12)
```

Comment:
The test as it is is broken (you can try running it yourself). We should probably come up with a better assertion.